tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -932,7 +934,10 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
 
             m(s"$testRoutePath/$path") ~> Route.seal(routes(creds)) ~> check {
               status should be(statusCode)
-              headers shouldBe List(RawHeader("Set-Cookie", "a=b"))
+              headers.exists(header => {
+                header.is("set-cookie") && header.value() == "a=b"
+              }) shouldBe true
+              headers.exists(_.is(ActivationIdHeaderInLowercase)) shouldBe true
```

Comment:
See comment above, please rewrite both of these into "contains" checks.

Reply:
got it